### PR TITLE
Fix cannot initialize pd if using ipv6 address

### PR DESCRIPTION
**Pull request by birdstorm**
Previously: if the pd is using an ipv6 address, HostAndPort cannot resolve address correctly.

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/TiSession.java [hunk 1]
```diff
@@ -110,17 +110,17 @@ public synchronized RegionManager getRegionManager() {
   public synchronized ManagedChannel getChannel(String addressStr) {
     ManagedChannel channel = connPool.get(addressStr);
     if (channel == null) {
-      HostAndPort address;
+      URI address;
       try {
-        address = HostAndPort.fromString(addressStr);
+        address = URI.create("http://" + addressStr);
```
Comment (zhexuany) [inline tikv-client/src/main/java/com/pingcap/tikv/TiSession.java]: Maybe you can move it to the field variable?

Comment (birdstorm) [reply tikv-client/src/main/java/com/pingcap/tikv/TiSession.java]: fixed in #589

**Pull request merged**